include/queue_fn.php

```diff
@@ -24,26 +24,20 @@ function remove_queue_item($id) {
  */
 function was_recently_delayed($cid) {
 
-	$was_delayed = false;
+	$r = q("SELECT `id` FROM `queue` WHERE `cid` = %d 
+		and last > UTC_TIMESTAMP() - interval 15 minute limit 1",
+		intval($cid)
+	);
+	if(dbm::is_result($r))
+		return true;
 
 	// Are there queue entries that were recently added?
 	$r = q("SELECT `id` FROM `queue` WHERE `cid` = %d
 		AND `last` > UTC_TIMESTAMP() - interval 15 minute LIMIT 1",
 		intval($cid)
 	);
 
-	$was_delayed = dbm::is_result($r);
```

Comment:
Why was this left out? Can it be that this pull request is missing some commits?